stream/popularity_mapper.js

```diff
@@ -0,0 +1,209 @@
+/**
+  The popularity mapper is responsible for generating a 'popularity'
+  value by inspecting OSM tags.
+
+  Disused and abandoned places are given a strong negative score.
+  If the popularity score is less than zero then the document is discarded.
+
+  Feel free to make changes to this mapping file!
+**/
+
+const through = require('through2');
+const peliasLogger = require('pelias-logger').get('openstreetmap');
+
+const config = {
+  // https://taginfo.openstreetmap.org/keys/importance
+  importance: {
+    international: { _score: 50000 },
+    national: { _score: 10000 },
+    regional: { _score: 5000 },
+    urban: { _score: 1000 },
+    suburban: { _score: 500 },
+    local: { _score: 100 },
+  },
+
+  // concordances
+  wikipedia: { _score: 3000 },
+  wikidata: { _score: 3000 },
+
+  // properties found on major landmarks &
+  // public buildings.
+  architect: { _score: 5000 },
+  heritage: { _score: 5000 },
+  'heritage:operator': { _score: 2000 },
+  historic: {
+    building: { _score: 5000 },
+    church: { _score: 5000 },
```

Comment:
historic=monument with a higher score that default?

Reply:
I saw a few cases where some old stone markers at the top of hills were marked `historic:monument` so I'm not so sure about it?

Reply:
Yes you are right, this is not as relevant as I thought. (I saw a lot of `historic=monument` + `name=*` in Paris and its region.)